etc/spark-jobserver.conf

```diff
@@ -0,0 +1,18 @@
+spark {
+  master = "local[4]"
+
+  job-number-cpus = 4
```

Comment:
Yeah, this was pretty much just taken from the defaults, but was also meant to line up with the `master` setting above it. https://github.com/azavea/docker-spark-jobserver/issues/4 is open to evaluate these settings more closely.